static/css/training.css

```diff
@@ -0,0 +1,95 @@
+#hero {
```

Comment:
A variety of indentation issues in this file. Make sure to standardize to 2 spaces.